src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Try to avoid using generic variable names like `result`. It doesn't provide any information about what this variable is used for. A more descriptive name can make your code easier to read and understand. For example, you could use `cssStylesObject` instead.